views/js/front.js

```diff
@@ -51,16 +51,16 @@ $(window).ready(function () {
 
         // Check if the viewport is set, else we gonna set it if we can.
         if (!$svg.attr('viewBox') && $svg.attr('height') && $svg.attr('width')) {
-          $svg.attr('viewBox', '0 0 ' + $svg.attr('height') + ' ' + $svg.attr('width'))
+          $svg.attr('viewBox', '0 0 ' + $svg.attr('height') + ' ' + $svg.attr('width'));
         }
 
         // Replace image with new SVG
         $img.replaceWith($svg);
 
         // set color of svg at load
+        // @todo: psr_icon_color is not set ?
```

Comment:
@Progi1984 Any idea of why there is this ?

Reply:
It is defined in https://github.com/PrestaShop/blockreassurance/blob/dev/blockreassurance.php#L500

Reply:
But it's a smarty variable 🤔 how can the JS read it ?